policies/secure-development-lifecycle.md

```diff
@@ -0,0 +1,45 @@
+# Secure Development Lifecycle Policy
+
+## Purpose
+The secure development lifecycle describes the approach that is used to define, develop, deliver and maintain software. 
+
+## Scope
+This lifecycle is intended to be used by software teams including product managers, designers, developers and quality assurance to develop secure software products. This policy describes the lifecycle of feature development.
+
+## Policy
+This lifecycle integrates security into all aspects of application software development which is comprised of six phases. 
+
+1. Requirements Gathering and Planning:
+  - Identify the scope, goals, and necessary resources for development
+  - Define functional and security requirements for the feature 
+    - Considering [core engineering principles](https://github.com/MetaMask/contributor-docs/blob/main/policies/engineering-principles.md)
+    - Security controls such as authentication, data encryption, and secure communication protocols
+
+2. Design:
+- Create a feature description that incorporates identified security controls and meets [definition of ready](https://www.notion.so/What-is-the-Definition-of-Ready-fa1df34b96734791be1293d4f536869c)
+- Perform threat modeling to identify potential risks and design appropriate countermeasures
+- Document design decisions, security controls, implementation and verification
+
+3. Development:
+- Adhere to [coding standards and secure coding practices](https://github.com/MetaMask/contributor-docs/tree/main/guides)
+- Implement features and security controls according to the design
+- Conduct code review for all changes
+- Ensure automated code quality and security testing checks are passing
+
+4. Testing:
+- Perform testing to validate the application meets desired requirements. The testing methodology is set by project specific guidelines. 
```

Comment:
We should specify what type of testing. Maybe change the first sentence to "Perform functional and non-functional testing to verify and validate that the application satisfies the required criteria."

Reply:
Okay, that is a fair point.